sqlc-database/hello/store/queries/queries.sql

```diff
@@ -0,0 +1,9 @@
+-- name: InsertGreeting :one
```

Comment:
This isn't inserting a greeting but rather it's incrementing the meeting count. How about naming it `IncrementMeetingCount`?